scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -436,5 +437,116 @@ public static BuildTargetResult TriggerDockerHubBuilds(BuildTargetContext c)
             }
             return c.Success();
         }
+
+        private const string PackagePushedSemaphoreFileName = "packages.pushed";
+
+        [Target(nameof(PrepareTargets.Init), nameof(InitPublish))]
+        public static BuildTargetResult PullNupkgFilesFromBlob(BuildTargetContext c)
+        {
+            Directory.CreateDirectory(Dirs.PackagesNoRID);
+
+            var hostBlob = $"{Channel}/Binaries/";
+
+            string forcePushBuild = Environment.GetEnvironmentVariable("FORCE_PUBLISH_BLOB_BUILD_VERSION");
+
+            if (!string.IsNullOrEmpty(forcePushBuild))
+            {
+                Console.WriteLine($"Forcing all nupkg packages for build version {forcePushBuild}.");
+                DownloadPackagesForPush(hostBlob + forcePushBuild);
+                return c.Success();
+            }
+
+            List<string> buildVersions = new List<string>();
+
+            Regex buildVersionRegex = new Regex(@"Binaries/(?<version>\d+\.\d+\.\d+-[^-]+-\d{6})/$");
```

Comment:
After the shared framework ships in june this regex may break. 

Recommend making the prerelease portion an optional group:
i.e `Binaries/(?<version>\d+\.\d+\.\d+([^-]+-\d{6})?)/$`